tests/js-tests/src/CocosDenshionTest/CocosDenshionTest.js

```diff
@@ -231,7 +231,11 @@ var soundId = null;
 
 var playMusic = function () {
     cc.log("play background music");
-    audioEngine.playMusic(MUSIC_FILE, false);
+    var musicFile = MUSIC_FILE;
+    if (cc.sys.os == cc.sys.OS_ANDROID) {
+        musicFile = "res/"+musicFile;
+    }
+    audioEngine.playMusic(musicFile, false);
```

Comment:
But why other resources, such as .png file don't have to add prefix `res`? For example, all the file path in [test_resources.js](https://github.com/cocos2d/cocos2d-x/blob/v3/tests/js-tests/src/tests_resources.js) don't need to add prefix `res`?